Core/Src/porting/lib/lz4_depack.c

```diff
@@ -1,10 +1,103 @@
-//#include <string.h>
+
```

Comment:
Please add a comment where the original source code came from, i.e. https://github.com/jibsen/blz4/blob/master/lz4_depack.c probably

Reply:
done. 
Thanks for the review and all the time you spent.